build.sbt

```diff
@@ -55,7 +55,7 @@ lazy val scalatestSettings = Seq(
   testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-oS"),
   logBuffered in testOnly in Test := false,
   libraryDependencies ++= Seq(
-    "org.scalatest"  %% "scalatest"  % "3.0.1"  % "test",
+    "org.scalatest"  %% "scalatest"  % "3.0.5"  % "test",
```

Comment:
fwiw, latest 3.0.x version is 3.0.9

Reply:
There are some issues with 3.0.6+, see https://github.com/NetLogo/NetLogo/issues/1974.